src/Bundler.js

```diff
@@ -242,10 +247,20 @@ class Bundler extends EventEmitter {
 
         this.entryAssets = new Set();
         for (let entry of this.entryFiles) {
-          let asset = await this.resolveAsset(entry);
-          this.buildQueue.add(asset);
-          this.entryAssets.add(asset);
+          try {
+            let asset = await this.resolveAsset(entry);
+            this.buildQueue.add(asset);
+            this.entryAssets.add(asset);
+          } catch (err) {
+            throw new Error(`Failed to load entrypoint: ${entry}!`);
```

Comment:
What do you think of a more precise error message, something like: `Failed to resolve entrypoint "${entry}" in "${rootDir}"`

Given that we infer `rootDir` I think it'd be better to give users more info so they can debug why we can't find their entrypoint.

Reply:
Sure thing, I'll update it